### ISSUE TITLE
Multipass snap edge channel has not been updated since March 19

### ISSUE THREAD

**Issue by holta** [canonical/multipass#3480]
[@EMG70 volunteered to test] Multipass 1.14.x (edge channel) on Ubuntu Server 24.04 (latest pre-release, installed just today) and `multipass launch` _does work_ right after boot.

[However, the] `multipass set local.driver=lxd` command does not work for him, even after `snap install lxd` and rebooting multiple times [...].  The error he gets is:

```
set failed: You have to `multipass set local.bridged-network=<name>` to use the bridged shortcut.
```

_Above Originally posted by @holta in https://github.com/canonical/multipass/issues/3464#issuecomment-2047930499_

The error message appears unhelpful, as it suggests running the following, _which does not work:_ 😢

```
root@ubuntu2404:~# multipass set local.bridged-network=eno1
set failed: You have to `multipass set local.bridged-network=<name>` to use the bridged shortcut.
```

Original failure, right after he installed Ubuntu Server 24.04:

![image](https://github.com/canonical/multipass/assets/2458907/0ce482b4-3f2c-4941-b8a2-ab441558e64d)

![image](https://github.com/canonical/multipass/assets/2458907/70f78ef8-5253-4cac-ad20-7f5b7b4bd930)

**Comment by townsend2010** [comment]
Hi @holta,

Yes, this issue was fixed recently via https://github.com/canonical/multipass/pull/3449.  It seems the version of the snap on this machine needs to be refreshed.  Please ensure the latest Multipass package from `edge` is installed.

**Comment by townsend2010** [comment]
For confirmation, could you please give the output of:
`$ multipass version`
and
`$ snap info multipass`

Thanks!

**Comment by townsend2010** [comment]
That looks to be the correct version, but I would like confirmation of `multipass version` as well please.

**Comment by EMG70** [comment]
![IMG_3304](https://github.com/canonical/multipass/assets/52469992/082f5591-0291-478a-9177-35655a2587d8)

**Comment by townsend2010** [comment]
Oh, it looks like something is wrong with how our CI is uploading to the `edge` channel- it is wayyyyy out of date:
```
latest/edge:      1.14.0-dev.1600+g2701b66b        2024-03-19 (12063)  67MB
```

**March 19, 2024**!!!
I will have to investigate our CI on why this hasn't been updated in so long.

**Comment by townsend2010** [comment]
Ok, after much investigation and working with the Launchpad Team (thanks @sn3rd!), I have found the problem (at least the first and hopefully only layer of the onion).  I will be working on a fix.  Thanks!

**Comment by holta** [comment]
Personal Opinion: Overall Multipass QA can greatly benefit if Multipass's CI posts "edge channel" builds for Windows and macOS too! 😄 

As discussed Jan 5 — might this be doable in 2024?

- https://github.com/canonical/multipass/issues/3346#issuecomment-1879094428

**Comment by townsend2010** [comment]
Hey @holta,

I agree and it is still on my TODO to figure out a way that we can do that which has visibility in discovering these packages.  We do currently build these, but it's not (easily) discoverable and will require some automation for them to be discoverable.

**Comment by holta** [comment]
> not (easily) discoverable and will require some automation for them to be discoverable.

Partially discoverable downloadable images might be perfectly sufficient initially?

(Perhaps advertised as "at your own risk" builds for developers only, helping Multipass to tighten QA feedback loops across more platforms?)

Thanks @townsend2010 for probing what's possible! 💯

**Comment by townsend2010** [comment]
Well, what I mean is that currently the artifacts are published publicly to an S3 bucket.  Without some manual intervention from the team to post the URL somewhere, one would have to "guess" the URL to get the artifacts.  Manually posting a URL every time there is a new build for the Windows and macOS platforms is very inefficient at best (and just plain wouldn't be done like now at worst) and I would rather automate that somehow.  But yes, there is definitely value in doing this.

**Comment by holta** [comment]
@EMG70

With Multipass 1.14.0-dev.1729+g8cbcc013 just released in recent minutes (thanks to PR #3481!) can you re-test this ticket's original `multipass set local.driver=lxd` failures that have been so aggravating in recent weeks? :pray:

```
# snap info multipass
...
channels:
  latest/stable:    1.13.1                    2024-04-01 (12146) 122MB -
  latest/candidate: 1.13.1                    2024-04-11 (12214) 122MB -
  latest/beta:      1.13.1                    2024-02-12 (11732) 122MB -
  latest/edge:      1.14.0-dev.1729+g8cbcc013 2024-04-11 (12223)  66MB -
installed:          1.14.0-dev.1600+g2701b66b            (12063)  67MB -

# snap refresh multipass
multipass (edge) 1.14.0-dev.1729+g8cbcc013 from Canonical✓ refreshed

# multipass version
multipass   1.14.0-dev.1729+g8cbcc013
multipassd  1.14.0-dev.1729+g8cbcc013
```

**Comment by EMG70** [comment]
Thank you all and  @townsend2010 ,I have tested and I can confirm all is working as before.I can set up a bridged network using the  refreshed Multipass edge.
